manage_data/manage_books.rb

```diff
@@ -0,0 +1,69 @@
+require_relative '../includes'
+require 'json'
+
+class BookHandle
+  def initialize
+    @books = if File.exist?('./data/books.json')
+               JSON.parse(File.read('./data/books.json'), object_class: OpenStruct)
+             else
+               []
+             end
+  end
+
+  def list_books
+    if @books.length.positive?
+      @books.each_with_index do |book, index|
+        puts "[#{index + 1}]"
+        puts "Publisher: #{book.Publisher}"
+        puts "Cover State: #{book.State}"
+        puts "Publish Date: #{book.Date}"
+        puts
+      end
+    else
+      puts 'The book list is empty. Create a new book!'
+    end
+  end
```

Comment:
- [ ] Kindly fix this issue when I try to add a new book and after that list all books, I got the following error.
- [ ] Also kindly fix this issue for when I try to add music albums and then try to list them, I got an error also.
>![image](https://user-images.githubusercontent.com/53402600/196948265-09491684-20cc-4426-943e-76319e2bd2e6.png)

Reply:
This change has been done well. I can now be able to add a book and list all books... I can also be able to create an album and list all albums. Great work